Index: yosys/Makefile
===================================================================
--- yosys.orig/Makefile	2014-09-05 22:18:08.594546002 +0200
+++ yosys/Makefile	2014-09-05 22:19:16.926546735 +0200
@@ -233,20 +233,20 @@
 yosys-abc: abc/abc-$(ABCREV)
 	$(P) cp abc/abc-$(ABCREV) yosys-abc
 
-test: $(TARGETS) $(EXTRA_TARGETS)
-	+cd tests/simple && bash run-test.sh
-	+cd tests/hana && bash run-test.sh
-	+cd tests/asicworld && bash run-test.sh
-	+cd tests/realmath && bash run-test.sh
-	+cd tests/share && bash run-test.sh
-	+cd tests/fsm && bash run-test.sh
-	+cd tests/techmap && bash run-test.sh
-	+cd tests/memories && bash run-test.sh
-	+cd tests/various && bash run-test.sh
-	+cd tests/sat && bash run-test.sh
-	@echo ""
-	@echo "  Passed \"make test\"."
-	@echo ""
+#test: $(TARGETS) $(EXTRA_TARGETS)
+#	+cd tests/simple && bash run-test.sh
+#	+cd tests/hana && bash run-test.sh
+#	+cd tests/asicworld && bash run-test.sh
+#	+cd tests/realmath && bash run-test.sh
+#	+cd tests/share && bash run-test.sh
+#	+cd tests/fsm && bash run-test.sh
+#	+cd tests/techmap && bash run-test.sh
+#	+cd tests/memories && bash run-test.sh
+#	+cd tests/various && bash run-test.sh
+#	+cd tests/sat && bash run-test.sh
+#	@echo ""
+#	@echo "  Passed \"make test\"."
+#	@echo ""
 
 VALGRIND ?= valgrind --error-exitcode=1 --leak-check=full --show-reachable=yes --errors-for-leak-kinds=all
 
@@ -327,6 +327,6 @@
 -include kernel/*.d
 -include techlibs/*/*.d
 
-.PHONY: all top-all abc test install install-abc manual clean mrproper qtcreator
+.PHONY: all top-all abc install install-abc manual clean mrproper qtcreator
 .PHONY: config-clean config-clang config-gcc config-gcc-4.6 config-gprof config-sudo
 
