Index: yosys/Makefile
===================================================================
--- yosys.orig/Makefile	2014-09-05 22:19:51.746547108 +0200
+++ yosys/Makefile	2014-09-05 22:20:14.654547354 +0200
@@ -262,11 +262,11 @@
 	@echo "  Passed \"make vloghtb\"."
 	@echo ""
 
-install: $(TARGETS) $(EXTRA_TARGETS)
-	$(INSTALL_SUDO) mkdir -p $(DESTDIR)/bin
-	$(INSTALL_SUDO) install $(TARGETS) $(DESTDIR)/bin/
-	$(INSTALL_SUDO) mkdir -p $(DESTDIR)/share/yosys
-	$(INSTALL_SUDO) cp -r share/. $(DESTDIR)/share/yosys/.
+#install: $(TARGETS) $(EXTRA_TARGETS)
+#	$(INSTALL_SUDO) mkdir -p $(DESTDIR)/bin
+#	$(INSTALL_SUDO) install $(TARGETS) $(DESTDIR)/bin/
+#	$(INSTALL_SUDO) mkdir -p $(DESTDIR)/share/yosys
+#	$(INSTALL_SUDO) cp -r share/. $(DESTDIR)/share/yosys/.
 
 manual: $(TARGETS) $(EXTRA_TARGETS)
 	cd manual && bash appnotes.sh
