Index: yosys-0.2.0+/Makefile
===================================================================
--- yosys-0.2.0+.orig/Makefile	2014-04-12 19:30:08.659862767 +0200
+++ yosys-0.2.0+/Makefile	2014-04-12 21:44:51.699931454 +0200
@@ -40,7 +40,7 @@
 endif
 
 YOSYS_VER := 0.2.0+
-GIT_REV := $(shell git rev-parse --short HEAD || echo UNKOWN)
+GIT_REV := d18c10d
 OBJS = kernel/version_$(GIT_REV).o
 
 # set 'ABCREV = default' to use abc/ as it is
